src/AdminConsole/Services/ISharedBillingService.cs

```diff
@@ -0,0 +1,22 @@
+using Stripe;
+
+namespace Passwordless.AdminConsole.Services;
+
+public interface ISharedBillingService
+{
+    Task UpdateUsage();
+
+    Task UpdateStripe(string subscriptionItemId, int users);
+
+    Task ConvertFromFreeToPaidAsync(string customerId, string clientReferenceId, string subscriptionId);
```

Comment:
Can we standardize on the `Async` suffix? Either use it or not use it

Reply:
Yeah agreed. I just thought it might be too much at once.